desktop/main/startApp.ts

```diff
@@ -94,8 +95,9 @@ const startApp = (): AppStore => {
   const $wallet = new $.BehaviorSubject<Wallet | null>(null);
   const $walletPath = new $.BehaviorSubject<string>('');
   const $networks = new $.BehaviorSubject<Network[]>([]);
-  const $currentNetwork = currentNetwork($wallet, $networks);
   const $nodeConfig = new $.Subject<NodeConfig>();
+  const $runNodeBeforeLogin = $.of(app.getLoginItemSettings().openAtLogin);
```

Comment:
I think since we wrap up this flag into Observable, then it should behave like the value that could be changed in time. So you can make it a Subject and integrate with the code in `main.dev.ts` (load & update value).
Then you can also put it into `syncToRenderer` function and have real-time updates on the client-side "for free" without an additional event and handler `IS_AUTO_START_ENABLED_REQUEST`.

Reply:
We don't need it on the client side, because we're reading this value only in the case of autostart system step.